datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -646,7 +646,8 @@ CREATE TABLE dcsa_im_v3_0.transport_call (
     mode_of_transport_code varchar(3) NULL REFERENCES dcsa_im_v3_0.mode_of_transport (mode_of_transport_code),
     vessel_id uuid NULL REFERENCES dcsa_im_v3_0.vessel(id),
     import_voyage_id uuid NULL, -- Will add the reference later once Voyage is added,
-    export_voyage_id uuid NULL -- Will add the reference later once Voyage is added
+    export_voyage_id uuid NULL, -- Will add the reference later once Voyage is added
+    port_call_status_code char(4) NULL
```

Comment:
`varchar(4)`

(char has no advantages over varchar in postgres and only issues)

Reply:
I tried to keep it consistent with what we used for codes, I will change it if that's the case.

Reply:
Fair.  Personally, I see no reason to promote the use of char as it is "at best" just as effective as varchar.  But the clean up of that can be done in a separate commit.